apps/marssnow/marssnow.yml

```diff
@@ -0,0 +1,19 @@
+name: Marssnow
+description: 'Desktop fast blogs app(博客2.0-快速博客).'
+website: 'http://www.marssnow.com'
```

Comment:
Change the `http` to `https`. I'm assuming for security purposes, Electron does not allow HTTP links on their websites.

[![best-regards](https://img.shields.io/badge/Best%20regards-Sanjit-accff2?style=for-the-badge)](https://www.sanjit.wtf)

Reply:
Thanks for u and HashimotoYT  help me 💙